Neos.Cache/Classes/Backend/RedisBackend.php

```diff
@@ -490,7 +490,8 @@ private function getRedisClient(): \Redis
         try {
             $connected = false;
             // keep the assignment above! the connect calls below leaves the variable undefined, if an error occurs.
-            if (strpos($this->hostname, '/') !== false) {
+            if (strncmp($this->hostname, '/', 1) === 0) {
```

Comment:
idk im more familiar with `strpos`, and had to first think of what `strncmp` does ^^

```suggestion
            if (strpos($this->hostname, '/') === 0) {
```

https://3v4l.org/j8sOg#v8.1.23

Reply:
While not as readable, it is faster. Given the backend is used for caching, speed is a feature. 😇

Reply:
we are lucky to have soonish only to think of php 8 `str_starts_with` ^^